File: python/basic13/maxMinAvg.py
arr = [2, 3, 4, 5, -1]

def minMaxAvg(arr):
    max = float("-inf")
    min = float("inf")
    sum = 0
    for i in range(len(arr)):
        sum += arr[i]
        if arr[i] > max:
            max = arr[i]
        elif arr[i] < min:
            min = arr[i]
        avg = sum/len(arr)
    print("this is the "+str(max),(min),avg)

minMaxAvg(arr)

File: practice/basic13-2/maxMinAvg.js
arr = [3, 1, -6, -2, 10, 30, 15];

function maxMinAvg(arr) {
    let max = -Infinity;
    let min = Infinity;
    let sum = arr[0];
    for(let i = 0; i < arr.length; i++) {
        sum += arr[i]
        if(arr[i] > max) {
            max = arr[i]
        } else if(arr[i] < min) {
            min = arr[i]
        }
    }
    let avg = sum/arr.length;
    return console.log("Max: " + max + " Min: " + min + " Avg: " + avg)
}

maxMinAvg(arr);